docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -22,7 +22,7 @@ Some features are enabled by default. You can disable these feature by setting t
 | Feature toggle name                  | Description                                                                                                                                                                                                                  | Enabled by default |
 | ------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------ |
 | `disableEnvelopeEncryption`          | Disable envelope encryption (emergency only)                                                                                                                                                                                 |                    |
-| `publicDashboards`                   | Enables public access to dashboards                                                                                                                                                                                          | Yes                |
+| `publicDashboards`                   | [Deprecated] Public dashboards are now enabled by default; to disable them, use the configuration setting. This feature toggle will be removed in v11.0.0.                                                                   | Yes                |
```

Comment:
```suggestion
| `publicDashboards`                   | [Deprecated] Public dashboards are now enabled by default; to disable them, use the configuration setting. This feature toggle will be removed in a future release.                                                                   | Yes                |
```

Reply:
Just in case it's not removed in v11.0.

Reply:
I agree that there is always a risk that we don't remove it in v11 but I find "a future release" very vague. What about this:
```suggestion
| `publicDashboards`                   | [Deprecated] Public dashboards are now enabled by default; to disable them, use the configuration setting. This feature toggle will be removed in the next major version.                                                                   | Yes                |
```
So that if it doesn't make it to v11, it's still valid for v12 😅

Reply:
Ah, that works better! Thanks.